cmd/oras/internal/option/remote.go

```diff
@@ -45,16 +45,27 @@ import (
 	"oras.land/oras/internal/version"
 )
 
+const (
+	usernameFlag               = "username"
+	passwordFlag               = "password"
+	passwordFromStdinFlag      = "password-stdin"
+	identityTokenFlag          = "identity-token"
+	identityTokenFromStdinFlag = "identity-token-stdin"
+)
+
 // Remote options struct contains flags and arguments specifying one registry.
 // Remote implements oerrors.Handler and interface.
 type Remote struct {
 	DistributionSpec
-	CACertFilePath    string
-	Insecure          bool
-	Configs           []string
-	Username          string
-	PasswordFromStdin bool
-	Password          string
+	CACertFilePath         string
+	Insecure               bool
+	Configs                []string
+	Username               string
+	PasswordFromStdin      bool
+	Password               string
+	IdentityTokenFromStdin bool
+	identityToken          string
```

Comment:
We really don't need those as `--identity-token` is just an alias of `--password`.

Reply:
removed these variables after resolving #1301